modules/newGame.js

```diff
@@ -0,0 +1,29 @@
+const urlPOST = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+const newGame = () => {
+  fetch(urlPOST, {
+    method: 'POST',
+    body: JSON.stringify({
+      name: 'cool game',
+
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error('From NewGame POST: Network response was not ok');
+      }
+      return response.json(); // This returns a Promise that resolves with the JSON data
+    })
+    .then((data) => {
+      const gameId = data.result.split(' ');
+      return gameId[3];
+    })
```

Comment:
- [ ] The project requirement as shown below requires you to make use of JavaScript's `async & await` to make API calls as opposed to using the `.then` like you have done. I would kindly suggest you make the necessary adjustment to your functions by using `async & await`
- * ## Project requirement
![prr](https://github.com/Nelofarzabi/Leaderboard/assets/89055502/57aaa6e0-cfc8-4cf7-9c4f-4a78600c77ff)